### PR TITLE
M code changes to fix date display in cprs

### PR DESCRIPTION
**Pull request by shabiel**

### CODE REVIEW

File: Packages.csv [hunk 1]
```diff
@@ -1506,6 +1506,7 @@ KERNEL,Kernel,XU,3.05,FAILED ACCESS ATTEMPTS LOG,HOLIDAY,10
 ,,,8992.2,ALERT RECIPIENT TYPE,,
 ,,,8992.3,ALERT CRITICAL TEXT,,
 ,,,8994.5,REMOTE APPLICATION,,
+KOREA SPECIFIC MODIFICATIONS,Korea Specific Modifications,UKO,,,,
```
Comment (ChristopherEdwards) [inline Packages.csv:1509]: Wouldn't this be better categorized as i18n changes?

File: Packages/Order Entry Results Reporting/Routines/ORQ20.m [hunk 2]
```diff
@@ -88,6 +90,10 @@
  Q Y
  ;
 DATE(X) ; -- Return date formatted as 00/00/0000 00:00
+ I $G(DUZ("LANG"))>1 D  Q $$FMTE^XLFDT(X) ; OSE/SMH - date i18n date/time
+ . ; 12345678901234
```
Comment (ChristopherEdwards) [inline Packages/Order Entry Results Reporting/Routines/ORQ20.m]: What are these comments trying to tell me?

File: Packages/VA FileMan/Globals/0.85+LANGUAGE.zwr [hunk 3]
```diff
@@ -501,6 +501,9 @@ OSEHRA ZGO Export: LANGUAGE
 ^DI(.85,196,0)="KONGO^KG^KON^^^M^L"
 ^DI(.85,197,0)="KONKANI^^KOK^^^M^L"
 ^DI(.85,198,0)="KOREAN^KO^KOR^^^I^L"
+^DI(.85,198,20.2)="S:$G(%DT)'[""I"" %DT=$G(%DT)_""I"" G CONT^%DT"
+^DI(.85,198,"DD")="S Y=$$FMTE^UKOUTL(Y,$G(%F))"
```
Comment (ChristopherEdwards) [inline Packages/VA FileMan/Globals/0.85+LANGUAGE.zwr:505]: would a long term plan be having these $$FMTE be part of FileMan's i18n?

**Review by ChristopherEdwards — APPROVED** [review]

File: Packages.csv (hunk 1 above)
Comment (shabiel) [reply Packages.csv:1509]: UKO files are Korea specific. So the category is correct. I didn't think far enough of weather our changes should be part of Fileman or part of each locale's implementation, which is what you are getting at. I don't think George T will take in changes for each country.

File: Packages/Order Entry Results Reporting/Routines/ORQ20.m (hunk 2 above)
Comment (shabiel) [reply Packages/Order Entry Results Reporting/Routines/ORQ20.m]: This was for me to figure out where the seconds are in a Timson date.

File: Packages/VA FileMan/Globals/0.85+LANGUAGE.zwr (hunk 3 above)
Comment (shabiel) [reply Packages/VA FileMan/Globals/0.85+LANGUAGE.zwr:505]: As I mentioned before, I doubt that we can fold these into Fileman itself.

File: Packages/Order Entry Results Reporting/Routines/ORQ20.m (hunk 2 above)
Comment (ChristopherEdwards) [reply Packages/Order Entry Results Reporting/Routines/ORQ20.m]: if the comment is important to leave there can you add more to it to make that obvious?

File: Packages.csv (hunk 1 above)
Comment (ChristopherEdwards) [reply Packages.csv:1509]: I'm OK with that.

**Comment by shabiel** [comment]
I can remove it. That's no problem at all.

**Comment by shabiel** [comment]
Okay. Your comments have been addressed; and I just added %DT as I couldn't
do that yet yesterday (I was waiting for George T to give me his okay).

**Comment by ChristopherEdwards** [comment]
Still approved, looks good.

**Pull request merged**